torch/_dynamo/variables/torch.py

```diff
@@ -307,9 +310,24 @@ def call_function(
             return NullContextVariable(**options)
         elif self.value is torch.autograd._profiler_enabled:
             unimplemented("torch.autograd._profiler_enabled not supported yet")
+        elif self.unsupported_scalar_return() and not config.dynamic_shapes:
+            # wrap_fx_proxy can only support certain function types; functions that return scalars
+            # are not supported when dynamic shapes are not enabled.
+            unimplemented(
+                f"{self.value} returns a scalar and dynamic shapes is not enabled"
+            )
         elif self.value is torch.jit.annotate:
             assert len(args) == 2
             return args[1]
+        elif self.value is torch.backends.cudnn.is_acceptable:
+            # is_acceptable(tensor) returns true if
+            #   (a) tensor dtype/device are supported by cudnn
+            #   (b) cudnn is available
+            #   (c) some initialization has completed
+            # technically, it depends on some global state from (c) (torch.backends.cudnn.__cudnn_version)
+            return ConstantVariable(
+                torch.backends.cudnn.is_acceptable(*args, **kwargs), **options
```

Comment:
`*args, **kwargs` seems like the wrong thing to pass.  These will be VariableTrackers not pytorch/python types.

Reply:
`is_acceptable(tensor)` only needs to use `tensor.device.type` and `tensor.dtype` which are both available on the VariableTracker object; is there a better way to get these? I see there's `get_real_value()` but that seems like overkill if we just want `tensor.device.type` and `tensor.dtype`?

Reply:
I'd suggest:
1) Actually check the count and types match, People could do crazy stuff like: `torch.backends.cudnn.is_acceptable("nope", [False])`
2) If the arg is a tensor, we could pass in something like: `torch.tensor(0, dtype=..., device=...)` (assuming all that matters are the dtype/device)